src/inclusion.md

```diff
@@ -126,8 +133,7 @@ We have provision for people with hearing, vision and mobility requirements.
   <h3>Dietary requirements</h3>
 
   <p>
-    We aim to meet all dietary requirements.
-    You will be able to specify these when purchasing your ticket.
+    We aim to meet all dietary requirements and our lunch time dishes are clearly labelled.
```

Comment:
Maybe "We aim to meet a wide range of dietary requirements" instead of "We aim to meet all..."?  I can imagine people getting upset if we don't consider their specific requirement to be something we consider "all"